src/components/app.tsx

```diff
@@ -243,7 +243,7 @@ export class AppComponent extends BaseComponent<IProps, IState> {
                     <FixWidthTabPanel width={`${tabWidth}px`}>
                       <Code>
                         <SyntaxHighlighter>
-                          {js_beautify(code)}
+                          {js_beautify(code.replace(/endStep\(\)\;\n/g, "").replace(/startStep\(\'.*\'\)\;\n/g, ""))}
```

Comment:
⬆  We also sneak in this fix in this  PR  👍  

Lets add a cleanup Chore in Pivotal Tracker to extract these regexs into a more general function.